spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index page for a user', type: :feature do
+  let(:user) { create(:user) }
+  let!(:posts) { create_list(:post, 3, author: user) }
+  let!(:comments) { create_list(:comment, 3, post: posts.first) }
+
+  before do
+    visit user_posts_path(user)
+  end
```

Comment:
- [ ] Please kindly add the missing test for the post_index page as highlighted in the screenshot in accordance to the requirement. 🙏 

<img width="498" alt="Untitled" src="https://github.com/shm04/blogApp/assets/103386183/9db6ce0f-933c-40ca-b4ff-045dedf360b1">